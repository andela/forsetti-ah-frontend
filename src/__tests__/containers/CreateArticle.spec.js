import React from 'react';
import { shallow } from 'enzyme';
import CreateArticlePage from '../../containers/CreateArticle';

describe('Create Article Page', () => {
  it('Should render component successfully', async () => {
    const wrapper = await shallow(<CreateArticlePage />);
    expect(wrapper.length).toEqual(1);
  });

  test('Should maintain existing snapshot', () => {
    const wrapper = shallow(<CreateArticlePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
