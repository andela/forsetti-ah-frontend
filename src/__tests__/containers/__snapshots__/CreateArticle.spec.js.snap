// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Article Page Should maintain existing snapshot 1`] = `ShallowWrapper {}`;
